apps/marketing/src/app/pricing/components/ComparisonTable/ComparisonTable.tsx

```
@@ -0,0 +1,121 @@
+function Row({ row }: { row: ComparisonRow }) {
+	return (
+		<tr className="border-b border-border/60">
+			<td className="py-4 pr-4 text-sm text-foreground">
+				<div className="flex items-center gap-2">
+					<span>{row.label}</span>
+					{row.comingSoon && <ComingSoonBadge />}
+				</div>
+			</td>
+			{row.values.map((value, index) => (
+				<td
+					key={`${row.label}-${index}`}
```

Biome (2.4.11)

[error] Avoid using the index of an array as key property in an element.

(lint/suspicious/noArrayIndexKey)